daml-foundations/daml-ghc/daml-stdlib-src/DA/Math.daml

```diff
@@ -0,0 +1,178 @@
+-- Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+daml 1.2
+-- | Math - Utility Math functions for `Decimal`
```

Comment:
Can we get a note with respect to performance and precision. These are pretty precise but very slow?